modules/electrs.nix

```diff
@@ -66,13 +77,14 @@ in {
       preStart = ''
         mkdir -m 0770 -p ${cfg.dataDir}
         chown 'electrs:electrs' ${cfg.dataDir}
```

Comment:
Can you remove that? Seems even less useful now?

Reply:
Perhaps also move the `chown -R '${cfg.user}:${cfg.group}' ${cfg.dataDir}` right below `mkdir`.